src/components/icon/IconCheckbox.tsx

```diff
@@ -0,0 +1,33 @@
+import Svg, { Props as SvgProps } from '../svg/Svg';
+
+interface Props extends SvgProps {
+  isChecked?: boolean;
+}
+
+const IconCheckbox = ({ isChecked = false }: Props) => {
```

Comment:
```suggestion
interface Props extends SvgProps {
  isChecked?: boolean;
}

const IconCheckbox = ({ isChecked = false, ...rest }: Props) => {
```

`isChecked`만 사용하면 extends한 이유가 퇴색될 거 같아요!